src/css/about-us.css

```diff
@@ -0,0 +1,171 @@
+.about-section {
+}
+
+.about-title-container {
+  max-width: 335px;
+  margin-bottom: 40px;
+}
+
+@media only screen and (min-width: 768px) {
+  .about-title-container {
+    max-width: 704px;
```

Comment:
remove max-width
it is defined in global container

Reply:
+